### ISSUE TITLE
VC1 parameter and missing K-parameters

### ISSUE THREAD

**Issue by stellabelin** [pharmpy/pharmpy#577]
A VC1 parameter is created when setting to three transits, 1 peripheral compartment, and ZO elimination. The following code can be used to reproduce the error (for [this](https://github.com/pharmpy/pharmpy/blob/master/tests/testdata/nonmem/models/mox2.mod) model):

```
set_transit_compartments(model, 3, keep_depot=False)
set_peripheral_compartments(model, 1)
model.model_code
set_zero_order_elimination(model)
print(model.model_code)
```
This produces the NONMEM code: 
```
$PK
CLMM = THETA(7)
VC1 = THETA(6)
KM = THETA(5)
VP1 = THETA(4)
QP1 = THETA(3)
MDT = THETA(2)
VC = THETA(1)*EXP(ETA(1))
V = VC
K12 = 3/MDT
K23 = 3/MDT
K34 = 3/MDT
K45 = QP1/V
K54 = QP1/VP1
```
Omitting one of the calls to `model_code` instead leads to the transit compartments not being included:
```
set_transit_compartments(model, 3, keep_depot=False)
set_peripheral_compartments(model, 1)
set_zero_order_elimination(model)
print(model.model_code)
```
Produces this:
```
$PK
KM = THETA(6)
VP1 = THETA(5)
QP1 = THETA(4)
MDT = THETA(3)
CLMM = THETA(2)*EXP(ETA(1))
VC = THETA(1)*EXP(ETA(2))
V = VC
```
Finally, having `model_code` after setting the transits has the original error of VC1 and doesn't have all transits:
```
set_transit_compartments(model, 3, keep_depot=False)
model.model_code
set_peripheral_compartments(model, 1)
set_zero_order_elimination(model)
print(model.model_code)
```
And produces this:
```
$PK
CLMM = THETA(7)
VC1 = THETA(6)
KM = THETA(5)
VP1 = THETA(4)
QP1 = THETA(3)
MDT = THETA(2)
VC = THETA(1)*EXP(ETA(1))
V = VC
K12 = 3/MDT
K23 = 3/MDT
K34 = 3/MDT
```

**Comment by stellabelin** [comment]
Model version 1 and 3 should also have an eta on CLMM, right?

**Comment by stellabelin** [comment]
Also occurs with MM elimination

**Comment by stellabelin** [comment]
Fixed in 0ad1cdf1fe6f43721c6962603b1a31912e812df1
